### PR TITLE
Documentation/rhym 64 socket docs

### PR DESCRIPTION
**Pull request by Yonava**
Hey, a bit late to the party! This pull request is for adding comprehensive documentation for our web socket endpoints. Each entry contains the name of the endpoint, the payload, the sender and recipient + a short description to give the reason for it existing some context. I also did some minor touch ups to the socket.js server file to correct a few past mistakes in the comments

### CODE REVIEW

**Comment by Raidienaty** [comment]
Bro, why no description again????

**Comment by Raidienaty** [comment]
Please edit your PR and add the description.

File: docs/sockets.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Socket endpoints are split into 2 categories: <a href="#connection">Connection Endpoints</a> & <a href="#gameplay">Gameplay Endpoints</a>
+<br><br>
+Connection endpoints are endpoints that exist solely to facilitate the function of connecting clients together for the purpose of making gameplay possible
+<br><br>
+Gameplay endpoints, in the other hand, are endpoints that are used by each client during gameplay to send data like new prompts, which prompt another player votes for etc.
+
+<div id="connection"></div>
+
+## Connection Endpoints
```
Comment (Raidienaty) [inline docs/sockets.md:17]: Can we add the type of connection to communicate with each? (GET, PUT, POST, etc)

Comment (Yonava) [reply docs/sockets.md:17]: That is not applicable for these endpoints

File: docs/sockets.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+<b>Data in Payload</b> view: string
+<br>
+Used by host to tell that players and audience what component they should render
+
+### host-present
+<b>Sender</b> `Host Client` -> <b>Recipient</b> `Player Client`
+<br>
+<b>Data in Payload</b> none
+<br>
+Lets host send a confirmation to players in room to confirm that the host is connected
```
Comment (Raidienaty) [inline docs/sockets.md:38]: Correcting line to:
Allows host to send a confirmation to players in room to confirm that the host is connected

**Review by Raidienaty — APPROVED** [review]
Comments resolved, approved by me.

**Review by Joshua-Burdick — APPROVED** [review]

**Pull request merged**